dist/source/repo-creator/choose-repository.css

```diff
@@ -145,3 +148,24 @@
 	border-radius:6px;
 	float:right;
 }
+@media all and (max-width: 384px) and (min-width: 300px){
+ .choose-repository .data-link {
+  font-size: 12px !important;
+
+}
+
+}
+ @media all and (max-width: 408px) and (min-width: 385px){
+.choose-repository .data-link {
+font-size: 12px !important;
+
+}
+
+}
+@media all and (max-width: 479px) and (min-width: 409px){
+.choose-repository .data-link {
+font-size: 12px !important;
+
+}
+
+} 
```

Comment:
Can we remove these `font-size` changes?  I did a brief check and they don't appear to be necessary for the fix to work and I would rather not have multiple font sizes as the screen resizes.  The `text-overflow`, `padding-right` and `overflow` changes appear to have resolved the issue.